scripts/bert/convert_tf_model.py

```diff
@@ -29,12 +29,16 @@
 
 parser = argparse.ArgumentParser(description='Conversion script for Tensorflow BERT model')
 parser.add_argument('--model', type=str, default='bert_12_768_12',
-                    help='BERT model name. options are bert_12_768_12 and bert_24_1024_16')
-parser.add_argument('--tf_checkpoint_dir', type=str, required=True,
+                    help='BERT model name. options are bert_12_768_12 and bert_24_1024_16.'
+                         'Default is bert_12_768_12')
+parser.add_argument('--tf_checkpoint_dir', type=str,
```

Comment:
What's the reason for us to provide the tensorflow convertion script? I think it might be better to remove this script, since we have provided our own parameters.